scania-digital-design-system/tegel
# feat:popover menu item

**Describe pull-request**  
Added a component for the popover-menu-item.

Note: This does not remove the styling for the old popover-menu-items which were basically just `<li>`s with anchor tags inside of them.

**Solving issue**  
Fixes:  -

**How to test**  
1. Go to Popover Menu
2. Check the design of the Popover Menu and its items.
